### PR TITLE
[SM-1266] Review PHP docs

### PR DESCRIPTION
**Pull request by mzieniukbw**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/SM-1266

## 📔 Objective

Review PHP language docs.
Fixed some of the typos too that prevented the example to work correctly.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation
  team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: languages/php/INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+- PHP >= 8.0
+- FFI extension enabled in PHP configuration
+- Composer
+- Bitwarden SDK native library, which is expected in one of below locations, depending on the OS and architecture.
+  If you prefer to build SDK yourself, see [SDK README.md](../../README.md) for instructions.
+    - Windows x86_64: `/lib/windows-x64/bitwarden_c.dll`
+    - Linux x86_64: `/lib/linux-x64/libbitwarden_c.so`
+    - macOS x86_64: `/lib/macos-x64/libbitwarden_c.dylib`
+    - macOS aarch64: `/lib/macos-arm64/libbitwarden_c.dylib`
```
Comment (Thomas-Avery) [inline languages/php/INSTALL.md:17]: I might be misunderstanding this, or I lack the knowledge on how PHP stuff works.

Is this from the root of the file system? 

I don't believe macOS will let you make `/lib/`. 

This also seems weird for Windows.

Couple with the changes in `languages/php/src/BitwardenLib.php` could the original intent be to have a `lib` directory in project's working directory?

File: languages/php/src/SecretsClient.php [hunk 2]
```diff
@@ -36,7 +36,7 @@ public function get_by_ids(array $secret_ids): \stdClass
         $project_get_by_ids_request->ids = $secret_ids;
         $project_get_by_ids_request->validate();
         $secrets_command = new SecretsCommand();
-        $secrets_command->get_by_ids = $project_get_by_ids_request->jsonSerialize();
+        $secrets_command->getByIds = $project_get_by_ids_request->jsonSerialize();
```
Comment (Thomas-Avery) [inline languages/php/src/SecretsClient.php:39]: :question: Was this causing issues?

File: languages/php/INSTALL.md (hunk 1 above)
Comment (mzieniukbw) [reply languages/php/INSTALL.md:17]: You are right, i did not know you can't have custom directories in macOS. I will revert this change.
Also i did test it on windows and the `/` is equivalent to current workspace drive letter, usually `C:\`. I will add this to the documentation.

File: languages/php/src/SecretsClient.php (hunk 2 above)
Comment (mzieniukbw) [reply languages/php/src/SecretsClient.php:39]: The PHP schema defines it's `getByIds`, so i technically fixing the fixing the PHP language client for the `$bitwarden_client->secrets->get_by_ids`, which before this change would just throw error and it works fine now  - also part of `example.php`.
If you want i can revert it and can fix it as separate issue.

Comment (Thomas-Avery) [reply languages/php/src/SecretsClient.php:39]: Thank you for the clarification, I see what's going on here now. 
I'm good with the change.

File: languages/php/INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  If you prefer to build SDK yourself, see [SDK README.md](../../README.md) for instructions.
+    - Windows:
+      - x86_64: `C:\lib\windows-x64\bitwarden_c.dll`.
+      - The drive letter is determined by the workdir of the PHP process, usually `C:`
+    - Linux:
+      - x86_64: `/lib/linux-x64/libbitwarden_c.so`
+    - macOS:
+      - x86_64: `./src/lib/macos-x64/libbitwarden_c.dylib`
+      - aarch64: `./src/lib/macos-arm64/libbitwarden_c.dylib`
+      - Where the `./src` is relative path to the [src](./src) directory.
```
Comment (Thomas-Avery) [inline languages/php/INSTALL.md:22]: Based on what the workflow for this used to look like, https://github.com/bitwarden/sdk/blob/8ea2f79c6bdddbfb1a6fa051d0461a1a9fb724c5/.github/workflows/publish-php.yml#L59-L71. 
I'm assuming the intent was to have everything be `./src/lib/<os>/<libbitwarden>`.

My suggestion would be to update `languages/php/src/BitwardenLib.php` and have each OS target have similar behavior.

**Review by Thomas-Avery — CHANGES_REQUESTED** [review]

**Review by Thomas-Avery — APPROVED** [review]
LGTM.

If @dani-garcia or @Hinton have more context on the original intent of where this should look for the native libs, I'm open to changes.

**Review by dani-garcia — APPROVED** [review]
No concerns from my side, I think looking for the libs with a relative path makes sense

**Pull request closed without merging**